slam-ws/src/nav_rtab/config/rtab_config.yaml

```diff
@@ -0,0 +1,17 @@
+subscibe_depth: true
+subscribe_laserScan: true
+frame_id: /base_link
+queue_size: 30
+
+# Deals with the "large angular variance error". Loop closure would essentially fail and map will be useless
+# Hard coded solution from rtab creator. Tune later
+
+odom_frame_id: odom
+odom_tf_linear_variance: 0.0001    # adjust depending on real odom error
+odom_tf_angular_variance: 0.0005   # adjust depending on real odom error
+
+Vis:
+  MinInliers: 12
+
+RGBD:
+  CreateOccupancyGrid: true
```

Comment:
**NOTE:** make sure that the Navigation Stack is subscribed to the correct `/map` topic (I believe the occupancy is not published under `/map` for rtab unlike gmapping)